canonical-web-and-design/ubuntu.com
# Build advantage from copy doc

## Done
Build the basic templates for logged out and logged in on advantage page.

_This is pre design, just using Vanilla and the copy docs_

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/advantage
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check that the content and flow matches [the copy doc](https://docs.google.com/document/d/1AW8wQu1Uvq0xFtd_c2JwNdic075MPOTx7k2KHqFPPeo/edit#heading=h.4roedpdi37ab)
- Sign in should work
- When signed in it should display your name and email
- Check that the show instructions interaction works

## Issue / Card
Fixes https://github.com/canonical-web-and-design/base-squad/issues/706
